### ISSUE TITLE
Add compatibility for react-router-dom 7

### ISSUE THREAD

**Issue by Sax388** [JesusTheHun/storybook-addon-remix-react-router#79]
**Is your feature request related to a problem? Please describe.**
When upgrading to `react-router-dom@"7.0.1"` the storybook addon complains because it depends on `peer react-router-dom@"^6.4.0" from storybook-addon-remix-react-router@3.0.1`.

**Describe the solution you'd like**
Add support for `react-router-dom` version `7`.

**Describe alternatives you've considered**
Stay on version `6` for now.

**Additional context**
```sh
npm error code ERESOLVE
npm error ERESOLVE could not resolve
npm error
npm error While resolving: storybook-addon-remix-react-router@3.0.1
npm error Found: react-router-dom@7.0.1
npm error node_modules/react-router-dom
npm error   react-router-dom@"7.0.1" from the root project
npm error
npm error Could not resolve dependency:
npm error peer react-router-dom@"^6.4.0" from storybook-addon-remix-react-router@3.0.1
npm error node_modules/storybook-addon-remix-react-router
npm error   dev storybook-addon-remix-react-router@"3.0.1" from the root project
npm error
npm error Conflicting peer dependency: react-router-dom@6.28.0
npm error node_modules/react-router-dom
npm error   peer react-router-dom@"^6.4.0" from storybook-addon-remix-react-router@3.0.1
npm error   node_modules/storybook-addon-remix-react-router
npm error     dev storybook-addon-remix-react-router@"3.0.1" from the root project
```

**Comment by JesusTheHun** [comment]
Hey @Sax388 👋 thanks for the heads up. I've released `v3.0.2` that allows for v7
Can you check it up ?

**Comment by Sax388** [comment]
Wow, that was quick. Thanks a million for the swift release. I've tested it with my story and I was routed just the way I expected to be 😃!
